Documentation/security/network/proxy/envoy.rst

```diff
@@ -16,6 +16,49 @@ for the cluster. Cilium proxy is distributed within the Cilium images.
 
 For more information on the version compatibility matrix, see `Cilium Proxy documentation <https://github.com/cilium/proxy#version-compatibility-matrix>`_.
 
+***********************
+Deployment as DaemonSet
+***********************
+
+Background
+==========
+
+When Cilium L7 functionality (Ingress, Gateway API, Network Policies with L7 functionality, L7 Protocol Visibility)
+is enabled or installed in a Kubernetes cluster, the Cilium agent starts an Envoy proxy as separate process within
+the Cilium agent pod.
+
+That Envoy proxy instance becomes responsible for proxying all matching L7 requests on that node.
+As a result, L7 traffic targeted by policies depend on the availability of the Cilium agent pod.
+
+With Cilium 1.14 it's possible to deploy the Envoy proxy as independently life-cycled DaemonSet called ``cilium-envoy``
+instead of running it from within the Cilium Agent Pod.
+
+The communication between Cilium agent and Envoy proxy takes place via UNIX domain sockets in both deployment modes.
+Be that streaming the access logs (e.g. L7 Protocol Visibility), updating the configuration via
+`xDS <https://www.envoyproxy.io/docs/envoy/latest/api-docs/xds_protocol>`_ or accessing the admin interface.
+
+Enable and configure Envoy DaemonSet
+====================================
+
+To enable the dedicated Envoy proxy DaemonSet, install Cilium with the Helm value ``envoy.enabled`` set to ``true``.
+
+Please see the :ref:`helm_reference` (Keys with ``envoy.*``) for detailed information on how to configure the Envoy proxy DaemonSet.
+
+Potential Benefits
+==================
+* Restarting Cilium Agent for upgrades does not impact live traffic proxied via Envoy.
+* Applying Envoy patch releases doesn't affect Cilium Agent.
+* Performance isolation between Envoy and Cilium Agent with the possibility to define separate CPU and memory limits.
+* Envoy application log doesn't get mixed with the one of the Cilium Agent.
+* Dedicated health probes for the Envoy proxy.
+* Explicit deployment of Envoy proxy during Cilium installation (compared to on demand in the embedded mode)
```

Comment:
Please make the list consistent (with verbs in all items, or none; and with consistent punctuation at the end of the items).

Let's also use `-` rather than `*` for item bullets. I think both work, but the rest of this doc uses dashes.

Reply:
@qmonnet thanks - i tried to come up with a more consistent update. PTAL :pray: